packages/block-library/src/latest-posts/edit.js

```diff
@@ -303,21 +303,35 @@ export default function LatestPostsEdit( { attributes, setAttributes } ) {
 								} )
 							}
 						/>
-						<BaseControl className="editor-latest-posts-image-alignment-control">
-							<BaseControl.VisualLabel>
-								{ __( 'Image alignment' ) }
-							</BaseControl.VisualLabel>
-							<BlockAlignmentToolbar
-								value={ featuredImageAlign }
-								onChange={ ( value ) =>
-									setAttributes( {
-										featuredImageAlign: value,
-									} )
-								}
-								controls={ [ 'left', 'center', 'right' ] }
-								isCollapsed={ false }
+						<ToggleGroupControl
+							className="editor-latest-posts-image-alignment-control"
+							__nextHasNoMarginBottom
+							label={ __( 'Image alignment' ) }
+							value={ featuredImageAlign }
+							onChange={ ( value ) =>
+								setAttributes( {
+									featuredImageAlign: value,
+								} )
+							}
+							isBlock
+						>
+							<ToggleGroupControlOption
+								value="none"
+								label={ __( 'None' ) }
+							/>
+							<ToggleGroupControlOption
+								value="left"
+								label={ __( 'Left' ) }
+							/>
+							<ToggleGroupControlOption
+								value="center"
+								label={ __( 'Center' ) }
+							/>
+							<ToggleGroupControlOption
+								value="right"
+								label={ __( 'Right' ) }
```

Comment:
Shouldn't these be `ToggleGroupControlOptionIcon` with the original icons? I guess we could change it to text options if that's better, but that's a separate discussion.

Reply:
I think we should keep the original icons too. You can also use the map function  ([Example](https://github.com/WordPress/gutenberg/blob/720522f3a86f78d5f34e8f364abda370054b6c27/packages/block-editor/src/layouts/constrained.js#L133-L144)).

Reply:
Thanks for the feedback. I have addressed this.